src/iexamine.cpp

```diff
@@ -3594,105 +3595,153 @@ void iexamine::climb_down( player &p, const tripoint &examp )
+            } //desktop case 1 rationing
+            case 2:{ //ammo type prefrences
+                add_msg("Not implemented yet.\n(╮°-°)╮┳━━┳ ( ╯°□°)╯ ┻━━┻");
+                //TODO: Menu assign prefrence (weight) to ammo types.  Prefrence to be refrenced by NPC choosing weapon
+                //Use relayering UI?
+            }
+            case 3:{ //crafting priorities
+                add_msg("Not implemented yet.\n(╮°-°)╮┳━━┳ ( ╯°□°)╯ ┻━━┻");
+                //TODO: Menu assign priorities (weight) to craft recipies.  NPCs to refrence priorities when crafting.
+            }
+            case 4:{ //view status.
+                if (base.get_level == 1) iexamine::base_view(p, examp);
+                else iexamine::base_view_ele(p, examp);
+            }
+            case 5:{ //base planner
+                add_msg("Not implemented yet.\n(╮°-°)╮┳━━┳ ( ╯°□°)╯ ┻━━┻");
+            }
+            case 6:{ //personel managment
+                add_msg("Not implemented yet.\n(╮°-°)╮┳━━┳ ( ╯°□°)╯ ┻━━┻");
+            }
+        } //desktop switch
+        desktop.query();
+    } while (desktop.ret); //desktop loop
 }
 
 /**
  * View base status/information from a bulletin board. (limited info)
+ * 
+ * This is supposed to represent viewing stuff posted on a bulletin board so,
+ * displaying the info in a whole bunch of small frames like they are diffrent notes should
+ * be a future goal.
  */
 void iexamine::base_view( player &p, const tripoint &examp) {
-    add_msg("Not implemented yet.\n(╮°-°)╮┳━━┳ ( ╯°□°)╯ ┻━━┻");
+    auto &base = p->home;
+    //Window size values are currently arbitrary but window shouldn't be full screen if possible
+    //This function is in the source but no header refrence...
+    //static WINDOW_PTR board = output::create_popup_window( 40, 40, output::PopupFlags::PF_GET_KEY);
+    
+    //Simple implementation for now
+    std::list<std::string> basicInfo;
+    basicInfo.push_back( string_format( _("%s's Home Sweet Base%n"), p.get_name(), int len );
+    //TODO add len '~'s as an underline 
+    basicInfo.push_back( _("") );//add blank line.
+    basicInfo.push_back( string_format( _("Command System Level: %d"), base.get_level() ) );
+    basicInfo.push_back( string_format( _("Base Ocupency: %d (%d Max)", base.num_personnel(), base.get_max_pop() );
+    basicInfo.push_back( string_format( _("Open Bunks: %d"), base.num_bunks() ) );
+    basicInfo.push_back( string_format( _("Open Storage: %d"), base.num_storage_open()));
+    basicInfo.push_back( string_format( _("Communal Storage Locations: %d"), base.num_comm_storage() ) );
+    std::list<std::string> rationInfo;
+    rationInfo.push_back( _("Supply Rationing") );
+    rationInfo.push_back( _("~~~~~~~~~~~~~~~~") );
+    rationInfo.push_back( string_format( _("Food: %s"), base_home::ration_levels::base.get_food_ration() );
+    rationInfo.push_back( string_format( _("Water: %s"), base_home::ration_levels::base.get_food_ration() );
```

Comment:
What do you want to achieve with `base_home::ration_levels::`? Does `base.get_food_ration()` not work?

Also: all 4 lines print the same number (call the same function).

Reply:
Enume string printing and lazy copy pasting.

Reply:
ration levels is an enum... get food ration levels is an int... i'll look up enums and make this better